utils/db.ts

```diff
@@ -171,6 +171,112 @@ export async function getItemsSince(msAgo: number) {
   });
 }
 
+// Notification
+export interface Notification {
+  userId: string;
+  type: string;
+  userFromId: string;
+  userFromLogin: string;
+  originId: string;
+  originTitle: string;
+  // The below properties can be automatically generated upon item creation
+  id: string;
+  createdAt: Date;
+}
+
+export function newNotificationProps(): Pick<Item, "id" | "createdAt"> {
+  return {
+    id: crypto.randomUUID(),
+    createdAt: new Date(),
+  };
+}
+
+/**
+ * Creates a new notification in KV. Throws if the item already exists in one of the indexes.
+ *
+ * @example New notification creation
+ * ```ts
+ * import { newNotificationProps, createNotification } from "@/utils/db.ts";
+ *
+ * const notification: Notification = {
+ *   userId: "example-user-id",
+ *   type: "example-type",
+ *   userFromId: "example-user-from-id"
+ *   userFromLogin: "example-user-from-login"
+ *   originId: "example-origin-id"
+ *   originTitle: "example-origin-title"
+ *   ...newNotificationProps(),
+ * };
+ *
+ * await createNotification(notification);
+ * ```
+ */
+export async function createNotification(notif: Notification) {
```

Comment:
```suggestion
export async function createNotification(notification: Notification) {
```
We can use the whole `notification` variable name since we do it everywhere else.